### PR TITLE
complete game_flow

### PR DESCRIPTION
**Pull request by garry911128**
In the main menu, Click mouse left will go to InGameScene.
InGameScene if press B or G will go to bad end or good end.(its the testing inputManager)
InGameEnd if press mouse left will restart game(go to main menu)

### BTW, the video encoding is wrong, but the logic of code is correct

### CODE REVIEW

**Review by cuchaos — APPROVED** [review]

File: Assets/Scripts/KeyboardInput/GameEndInputManager.cs
Comment (kyynk) [inline Assets/Scripts/KeyboardInput/GameEndInputManager.cs:1]: add namespace `KeyboardInput`.

File: Assets/Scripts/KeyboardInput/GameStartInputManager.cs
Comment (kyynk) [inline Assets/Scripts/KeyboardInput/GameStartInputManager.cs:1]: add namespace `KeyboardInput`

File: Assets/Scripts/KeyboardInput/InGaneInputManager.cs
Comment (kyynk) [inline Assets/Scripts/KeyboardInput/InGaneInputManager.cs:1]: add namespace `KeyboardInput`

**Review by kyynk — CHANGES_REQUESTED** [review]

File: Assets/Scripts/KeyboardInput/GameEndInputManager.cs
Comment (garry911128) [reply Assets/Scripts/KeyboardInput/GameEndInputManager.cs:1]: fix

File: Assets/Scripts/KeyboardInput/GameStartInputManager.cs
Comment (garry911128) [reply Assets/Scripts/KeyboardInput/GameStartInputManager.cs:1]: fix

File: Assets/Scripts/KeyboardInput/InGaneInputManager.cs
Comment (garry911128) [reply Assets/Scripts/KeyboardInput/InGaneInputManager.cs:1]: fix

**Review by kyynk — APPROVED** [review]
LGTM

**Review by cuchaos — APPROVED** [review]

**Pull request merged**